docs/user_manual/sra.md

```diff
@@ -0,0 +1,153 @@
+
+{% assign subsub = subsub | plus: 1 %}
+{% include userman-heading.md section=section sub=sub subsub=subsub section-title=section-title title="Adding Studies" %}
+
+There are two ways to create a study:
+
+1. Go to the studies list and click the "Add" button in the toolbar at the top of the table.
+1. Go to the project page, expand the "Studies" section and click the "Add" button in the toolbar at the top of the table.
+
+A study can only contain information about samples from a single project. The
+other information will be included in the metadata sent to the ENA.
+
+{% assign subsub = subsub | plus: 1 %}
+{% include userman-heading.md section=section sub=sub subsub=subsub section-title=section-title title="Deleting Studies" %}
+
+Studies can be deleted from either the "Studies List" or the project page.
+Check the checkbox beside the study and click the "Delete" button from the
+toolbar.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Experiments" %}
+
+[An experiment](https://ena-docs.readthedocs.io/en/latest/prog_04.html) binds together a study, sequencing platform, and library.
+
+{% assign subsub = 1 %}
+{% include userman-heading.md section=section sub=sub subsub=subsub section-title=section-title title="Experiments List" %}
+
+Experiments can be seen in several places:
+
+* From the "SRA" section on the left, choose "Experiments" to view all
+  experiments. This shows all experiments.
+* From the "Tracking" section on the left, choose "Studies", then select the study
+  of interest. This shows experiments associated with this study.
+* From the "Tracking" section on the left, choose "Sequencer Runs", then select the
+  run of interest. This shows experiments associated with this sequencing run.
+* From the "Tracking" section on the left, choose "Libraries", then select the
+  library of interest. This shows experiment associated with this library.
+
+{% assign subsub = subsub | plus: 1 %}
+{% include userman-heading.md section=section sub=sub subsub=subsub section-title=section-title title="Creating an Experiment" %}
+
+From the "Tracking" section on the left, choose "Sequencer Runs", and then
+select the sequencer run of interest. If pools have not already been assigned
+to lanes, assign them. From the "Experiment" table, choose "Create New".
+
+An experiment connects a study to a library on a sequencing platform. Select
+the appropriate library from the dialog box. A second dialog box will appear
+with all the studies that belong to the project of this library. Select the
+appropriate library and provide the title and alias of the experiment. The
+alias and title map directly to the alias and title in the [ENA
+model](https://ena-docs.readthedocs.io/en/latest/prog_04.html#create-the-run-and-experiment-xml).
+
+Once an experiment exists, all sequencer runs of this library from the same
+platform are connected with this experiment.
```

Comment:
This doesn't seem to be entirely accurate --  I made [this experiment|https://miso.gsi.oicr.on.ca/miso/experiment/2] for the library listed in the experiment title, but that library is also on another run which doesn't show up in the experiment. It does seem to capture all the sequencing lanes of this library from the same run + platform though... perhaps this is a MISO bug, if an SRA/ENA experiment is supposed to capture all runs for a given library?

Reply:
Docs are wrong. I will fix.